server/sonar-server/src/main/java/org/sonar/server/serverid/ws/ShowAction.java

```diff
@@ -73,14 +72,11 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkPermission(SYSTEM_ADMIN);
-    DbSession dbSession = dbClient.openSession(true);
-    try {
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not system administration... or is that the name we give to root?